scripts/gitlab/lingua.dic

```diff
@@ -125,6 +125,7 @@ iterable
 jaeger/MS
 js
 judgement
+judgements
```

Comment:
```suggestion
judgement/S
```
_should_ work in theory, but sometimes it doesn't :shrug: , in that case, ignore

Reply:
Thanks for the tip, I'll try.